templates/tableau-cluster-linux.template

```diff
@@ -0,0 +1,1539 @@
+                {
+                    "Ref":"SSLCertificateARN"
+                }
+            ]
+        },
+        "HasSSLCertificate":{
+            "Fn::Not":[
+                {
+                    "Condition":"NoSSLCertficate"
+                }
+            ]
+        },
+        "NoServerSecurityGroup":{
+            "Fn::Equals":[
+                "",
+                {
+                    "Ref":"ExistingSecurityGroup"
+                }
+            ]
+        },
+        "HasServerSecurityGroup":{
+            "Fn::Not":[
+                {
+                    "Condition":"NoServerSecurityGroup"
+                }
+            ]
+        },
+        "NoElasticIP":{
+            "Fn::Equals":[
+                "",
+                {
+                    "Ref":"IPAddress"
+                }
+            ]
+        },
+        "HasElasticIP":{
+            "Fn::Not":[
+                {
+                    "Condition":"NoElasticIP"
+                }
+            ]
+        },
+        "NoDNSEntry":{
+            "Fn::Or":[
+                {
+                    "Fn::Equals":[
+                        "",
+                        {
+                            "Ref":"AWSHostedZoneID"
+                        }
+                    ]
+                },
+                {
+                    "Fn::Equals":[
+                        "",
+                        {
+                            "Ref":"AWSPublicFQDN"
+                        }
+                    ]
+                }
+            ]
+        },
+        "CreateDNSEntry":{
+            "Fn::Not":[
+                {
+                    "Condition":"NoDNSEntry"
+                }
+            ]
+        },
+        "IsCentos":{
+            "Fn::Equals":[
+                "CentOS-7-HVM",
+                {
+                    "Ref":"AMIOS"
+                }
+            ]
+        }
+    },
+    "Resources":{
+        "VpcInfo":{
+            "Type":"Custom::VpcInfo",
+            "Properties":{
+                "ServiceToken":{
+                    "Fn::GetAtt":[
+                        "InfoLambda",
+                        "Arn"
+                    ]
+                },
+                "Region":{
+                    "Ref":"AWS::Region"
+                },
+                "Func":"DescribeVpc",
+                "VpcId":{
+                    "Ref":"VpcId"
+                }
+            }
+        },
+        "InfoLambda":{
+            "Type":"AWS::Lambda::Function",
+            "Properties":{
+                "Code":{
+                    "S3Bucket": "quickstart-reference",
+                    "S3Key": "tableau/server/latest/functions/packages/getVPCInfo/info_lambda.zip"
+                },
+                "Handler":"index.handler",
+                "Role":{
+                    "Fn::GetAtt":[
+                        "LambdaExecutionRole",
+                        "Arn"
+                    ]
+                },
+                "Runtime":"nodejs4.3",
```

Comment:
Worth moving to a newer version that's supported by Lambda, if you can.

Reply:
I didn't write this function. I am not sure if it works with a newer version.